### ISSUE TITLE
No toolbar and no View > Toolbar > PlantUML option to choose from

### ISSUE THREAD

**Issue by bokenator** [lodahl/Libo_PlantUML#19]
- Installed extension in LibreOffice version 6.1.5.2.
- Restarted LibreOffice after install
- No toolbar added, and no "PlantUML" option in View > Toolbar menu.

**Comment by lodahl** [comment]
Hi bokenator,
Please check if the extension is properly enabled (Tools - Extension Manager).
Otherwise please try to uninstall and install it again.
What operating system are you using(Mac/Windows/Linux)?

**Comment by bokenator** [comment]
Hi lodahl,
Thanks for the quick response!
I am on linux (Debian Buster).
I have just tried uninstalling and installing it again, but still no luck. Is there error logs anywhere or a way to print out the errors? Perhaps it will help you figure out where the issue is?

**Comment by lodahl** [comment]
Please go to Tools - Macros - Execute macro
Do you see PlantUML under "My macros" ?

**Comment by bokenator** [comment]
Yes, I do see PlanUML under my macros. But when i try to run it, it says "A scripting framework error occurred while running the Basic script PlantUML.PlantUML.CheckExtraOpen. Message: wrong number of paramters!"
